explainaboard_client/cli/delete_systems.py

```diff
@@ -19,14 +20,14 @@ def main():
     parser.add_argument(
         "--username",
         type=str,
-        default=explainaboard_client.username,
+        default=os.environ.get("EB_USERNAME"),
```

Comment:
This would expose the environment variable to the stdout. Manually checking the absense of the parameter is better. Same for other parts.

```python
parser.add_argument("--username")
args = parser.parse_args()
username = args.username if args.username is None else os.environ.get("EB_API_KEY")

Reply:
Thanks! Fixing it now